app/v2/notifications/post_notifications.py

```diff
@@ -201,14 +201,13 @@ def process_sms_or_email_notification(
         template_with_content=template_with_content
     )
 
-    if service.id in current_app.config.get('HIGH_VOLUME_SERVICE') and api_user.key_type == KEY_TYPE_NORMAL \
-       and notification_type == EMAIL_TYPE:
```

Comment:
We should make this strict so check that the notification type is in email or sms, just in case we get a high volume service who then decide to send a few letters

Reply:
This code path is only for SMS and EMAIL

Reply:
Ah yes, thanks. All good as is then.